airbyte-webapp/src/views/Connector/ConnectorForm/components/FrequentlyUsedConnectors/index.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+
+import { mockData } from "../../../../../test-utils/mock-data/mockFrequentlyUsedDestinations";
+import { FrequentlyUsedConnectorsCard } from "./FrequentlyUsedConnectorsCard";
+
+export default {
+  title: "Views/FrequentlyUsedConnectors",
+  component: FrequentlyUsedConnectorsCard,
+  args: {
+    destinations: mockData,
+    propertyPath: "serviceType",
+  },
+} as ComponentMeta<typeof FrequentlyUsedConnectorsCard>;
```

Comment:
This story doesn't seem to be rendering anything - is it missing some default args? https://63053d1047904393cf8d8faa-wolmvqrphe.chromatic.com/?path=/story/views-frequentlyusedconnectors--template

Reply:
Nice catch! I didn't check it 😢 
Will be fixed + improved in the next commit :)